application-load-balancer/lambda_function/secret_rotator.py

```diff
@@ -0,0 +1,625 @@
+import json
+import os
+import boto3
+import logging
+import requests
+import time
+#import uuid
+from typing import Tuple, Dict, Any, List, Optional
+from slack_service import SlackNotificationService
+from requests.exceptions import RequestException
+
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger()
```

Comment:
logger could be passed into the class when its instantiated in the handler instead of instantiating one here as well?

Reply:
Good suggestion, I've updated the code to follow dependency injection of logger